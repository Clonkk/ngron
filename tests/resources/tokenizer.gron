json = {};
json.integers = [];
json.integers[0] = 0;
json.integers[1] = 1;
json.integers[2] = -1;
json.integers[3] = 1231231434344;
json.floats = [];
json.floats[0] = 0.0;
json.floats[1] = 1.1;
json.floats[2] = -1.1;
json.floats[3] = 1231231434344.1231231434344;
json.floats[4] = 0.1231231434344;
json.floats[5] = -0.4e4;
json.floats[6] = 1.6E-12;
json.string = "simple string";
json.true = true;
json.false = false;
json.null = null;
