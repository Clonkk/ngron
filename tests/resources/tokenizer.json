{
    "integers": [
        0,
        1,
        -1,
        1231231434344
    ],
    "floats": [
        0.0,
        1.1,
        -1.1,
        1231231434344.1231231434344,
        0.1231231434344,
        -0.4e4,
        1.6E-12
    ],
    "string": "simple string",
    "true": true,
    "false": false,
    "null": null
}