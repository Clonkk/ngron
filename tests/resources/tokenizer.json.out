Token(LeftBrace, '{')
Token(String, 'integers')
Token(Colon, ':')
Token(LeftBracket, '[')
Token(Number, '0')
Token(Comma, ',')
Token(Number, '1')
Token(Comma, ',')
Token(Number, '-1')
Token(Comma, ',')
Token(Number, '1231231434344')
Token(RightBracket, ']')
Token(Comma, ',')
Token(String, 'floats')
Token(Colon, ':')
Token(LeftBracket, '[')
Token(Number, '0.0')
Token(Comma, ',')
Token(Number, '1.1')
Token(Comma, ',')
Token(Number, '-1.1')
Token(Comma, ',')
Token(Number, '1231231434344.1231231434344')
Token(Comma, ',')
Token(Number, '0.1231231434344')
Token(Comma, ',')
Token(Number, '-0.4e4')
Token(Comma, ',')
Token(Number, '1.6E-12')
Token(RightBracket, ']')
Token(Comma, ',')
Token(String, 'string')
Token(Colon, ':')
Token(String, 'simple string')
Token(Comma, ',')
Token(String, 'true')
Token(Colon, ':')
Token(Boolean, 'true')
Token(Comma, ',')
Token(String, 'false')
Token(Colon, ':')
Token(Boolean, 'false')
Token(Comma, ',')
Token(String, 'null')
Token(Colon, ':')
Token(Null, 'null')
Token(RightBrace, '}')
Token(Eof, '')